nuxeo-jsf/nuxeo-platform-webapp-base/src/main/resources/web/nuxeo.war/widgets/listing/listing_lock_icon_widget_template.xhtml

```diff
@@ -4,6 +4,6 @@
   xmlns:f="http://java.sun.com/jsf/core"
   id="#{widget.id}"
   rendered="#{not empty field and not empty field_1}">
-  <f:attribute name="title" value="#{messages['label.summary.by']} #{nxu:userFullName(field_1.owner)}, #{messages['label.summary.the']} #{field_1.created.time}" />
+  <f:attribute name="title" value="#{messages['label.summary.by']} #{nxu:userFullName(field_1.owner)}, #{messages['label.summary.the']} #{nxu:formatDateUsingFullDateAndTimeFormatter(field_1.created.time)}" />
   <f:attribute name="styleClass" value="tipsyShow" />
-</h:graphicImage>
+</h:graphicImage>
```

Comment:
Here you removed a terminating newline for no reason. I'm sorry to be insisting but a corollary of having patches reviewed by other is that you have to make the patches *minimal* so that there's a minimum of distraction for reviewers. Please review your complete diffs before creating the PR and remove any spurious changes not related to the code.